Schemaless (#143)
meilisearch/documentation: references/documents.md

<RouteHighlighter method="POST" route="/indexes/:uid/documents"/>

Add a list of documents or replace them if they already exist.

> If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **the whole old document**  will be override by the new document. Thus, removing any fields previously in the document not present in the new document.

Reviewer: ```suggestion
If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **the whole old document** will be overridden by the new document. Thus, removing any fields previously in the document not present in the new document.
```

Author: changed with `overwritten`

Author: Changed with `If you send an already existing document (same [id](/guides/main_concepts/documents.md#identifier)) the **whole existing document** will be overwritten by the new document. Fields previously in the document not present in the new document are removed.`

<RouteHighlighter method="PUT" route="/indexes/:uid/documents"/>

> Add a list of documents or update them if they already.

Reviewer: ```suggestion
Add a list of documents or update them if they already exist.
```

> If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **only the field of the new documents** will be changed in the old document. Thus, any fields not present in the new document remained unchanged.

Reviewer: ```suggestion
If you send an already existing document (same [identifier](/guides/main_concepts/documents.md#identifier)) **only the fields of this new document** will be changed in the old document. Thus, any fields not present in the new document remain unchanged.
```

Reviewer: I dont really understand the senteces. I guess that you try to describe the partial updating without mention it because I know MeiliSearch API (and because I can see it on the git diff).
I really think that the word "partial update" should stay, to be understood by most of the people. Anyone who used an API before know what a partial update is.
For those who don't know, indeed, it's ok to explain, and your explanation is welcome. But I would use more "field in the request body" than "fields of new document". I find it confusing.
Another point of view is needed :) @tpayet

Author: Do you like this more : 
`
If you send an already existing document (same [id](/guides/main_concepts/documents.md#identifier)) the old document will be only partially updated according to the fields of the new document. Thus, any fields not present in the new document are kept and remained unchanged.
`

> To completely overwrite a document, check out the [add and replace documents](/references/documents.md#add-or-replace-documents).

Reviewer: ```suggestion
To completely overwrite a document, check out the [add or replace documents](/references/documents.md#add-or-replace-documents).
```

Author: applied !
